### PR TITLE
App: Controls: Improve the vault view images

### PR DESCRIPTION
**Pull request by alistair23**
Make the buttons smaller and ensure the background is transparent.

The spacing between the buttons is still too large, but I'm not sure how to fix that.

Signed-off-by: Alistair Francis <alistair@alistair23.me>

### CODE REVIEW

File: src/App/Controls/LabeledValueCell.cs [hunk 1]
```diff
@@ -114,6 +114,23 @@ public class LabeledValueCell : ExtendedViewCell
...
             }
+            else if (Device.RuntimePlatform == Device.UWP)
+            {
+                buttonStackLayout.Spacing = 0;
+
+                if (Button1 != null)
+                {
+                    Button1.BackgroundColor = Color.Transparent;
+                    Button1.BorderWidth = 0;
+                    Button1.Scale = 0.5;
```
Comment (kspearrin) [inline src/App/Controls/LabeledValueCell.cs]: I don't think we want to scale the button. we just need to use the proper sized images as discussed earlier

Comment (alistair23) [reply src/App/Controls/LabeledValueCell.cs]: I was a little worried how that would look on high res screens, as UWP doesn't have dynamic image selection like Android and iOS (at least not with Xamarin), but I'm not sure how well scale will work either on high res screens.

**Comment by alistair23** [comment]
Fixed!

File: src/UWP.Images/UWP.Images.projitems [hunk 2]
```diff
@@ -9,8 +9,6 @@
     <Import_RootNamespace>UWP.Images</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Content Include="$(MSBuildThisFileDirectory)android.png" />
```
Comment (kspearrin) [inline src/UWP.Images/UWP.Images.projitems]: I believe there are cases when these could be invoked still

Comment (alistair23) [reply src/UWP.Images/UWP.Images.projitems]: Shouldn't it have given an error during build then?

Comment (kspearrin) [reply src/UWP.Images/UWP.Images.projitems]: It would be a runtime error if the image was trying to be displayed and the file does not exist.

Comment (alistair23) [reply src/UWP.Images/UWP.Images.projitems]: Ok, I can add it back. Just the Android one?

Comment (kspearrin) [reply src/UWP.Images/UWP.Images.projitems]: Android and apple are both used in cases when the URI uses their app's scheme

Comment (alistair23) [reply src/UWP.Images/UWP.Images.projitems]: Ok, I'll add those back in.

**Comment by alistair23** [comment]
Updated!

**Comment by alistair23** [comment]
Updated again with correct spacing

**Pull request merged**